source/Editor/PostBuildTriggerGA.cs

```diff
@@ -0,0 +1,39 @@
+using System.IO;
+using UnityEngine;
+using UnityEditor;
+using UnityEditor.Callbacks;
+using UnityEditor.iOS.Xcode;
```

Comment:
The namespace `UnityEditor.iOS.Xcode` is define in "unityEditor.iOS.Extensions.xcode.dll" which is only installed when you also have IOSSupport playbackEngine installed. So you will want to add `#if UNITY_IOS` guards.